pyproject.toml

```diff
@@ -66,7 +66,7 @@ test = [
     "pytest-sugar",
 ]
 dev = [
-    "black",
+    "black==19.10b0",
```

Comment:
I think we should add a comment on why this is pinned to this version so that future generations will know when this can be unpinned.

Reply:
I was getting one failing CI lane because of formatting lints. Lates Black changed formatting of few lines...